app/models/affiliate.rb

```diff
@@ -0,0 +1,18 @@
+class Affiliate < ApplicationRecord
+
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+
+  has_many :coordinators
+  has_many :tutors
+
+  has_many :matches
```

Comment:
It doesn't have to have matches. But it will be nice to allow the user see tutors and students relationships under each affiliate. What do you think?